### PR TITLE
adding auto tikz algebra to grammars + fixing whitespace to be compat…

### PR DESCRIPTION
**Pull request by sjanssen2**
…ible with Bellman's cafe

### CODE REVIEW

File: macrostate.gap [hunk 1]
```diff
@@ -26,9 +26,15 @@ include "Algebras/Probing/alg_probing_macrostate.gap" //an algebra for integrati
...
 algebra alg_enum auto enum ;
+algebra alg_tikz auto tikz;
+
 
 include "Grammars/gra_macrostate.gap"
 
+/*
+example inputs: CACUGUAAAGCUAACUUAGCAUUAACCUUUUAAGUUAAAGAUUAAGAGAACCAACACCUCUUUACAGUGACCA
+*/
+
```
Comment (crsct) [inline macrostate.gap:36]: should this be in the same commit? not that it matters greatly.

**Review by crsct — APPROVED** [review]
~LGTM~, the comment is minor, as it is only an added comment!

EDIT:
Seems I was a bit preemptive, as spacing seemed to be the issue, I ought to look at the files more closely, will comment again after checking them.

File: macrostate.gap (hunk 1 above)
Comment (sjanssen2) [reply macrostate.gap:36]: oh, I was not aware someone would read this :-) Thanks! This is special syntax to add a default input example when loaded on bellmanscafe.jlab.bio

**Review by crsct — APPROVED** [review]
besides the comment and the new PR #75 this LGTM

**Pull request merged**